libs/components/forms/src/lib/modules/field-group/field-group.component.ts

```diff
@@ -0,0 +1,55 @@
+import { CommonModule } from '@angular/common';
+import { Component, HostBinding, Input, booleanAttribute } from '@angular/core';
+
+/**
+ * Organizes form fields into a group.
+ */
+@Component({
+  selector: 'sky-field-group',
+  templateUrl: './field-group.component.html',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+
+      .sky-field-group-content {
+        margin-top: var(--sky-margin-stacked-sm);
+      }
+    `,
+  ],
+  standalone: true,
+  imports: [CommonModule],
+})
+export class SkyFieldGroupComponent {
+  /**
+   * The text to display as the field group's label.
+   * @preview
+   */
+  @Input({ required: true })
+  public labelText!: string;
+
+  /**
+   * Indicates whether to hide the `labelText`.
+   * @preview
+   */
+  @Input({ transform: booleanAttribute })
+  public labelHidden = false;
+
+  /**
+   * Whether the field group is stacked on another component. When specified, the appropriate
```

Comment:
Story mentions this (and Adam has said it in the past), but it should be "...is stacked on another field group". Just so we're precise with our wording/design recommendations.